recipe/meta.yaml

```diff
@@ -449,13 +454,14 @@ outputs:
       host:
         - {{ pin_subpackage("clang", exact=True) }}
         # Use the same requirements as the top-level requirements
-        - libcxx {{ cxx_compiler_version }}  # [osx]
+        - libcxx-devel {{ cxx_compiler_version }}   # [osx]
         - llvmdev =={{ version }}
         - libxml2
         - zlib
         - zstd
       run:
         - libstdcxx-devel_{{ target_platform }}  # [linux]
+        - libcxx-devel                           # [osx]
```

Comment:
We need to pin the version here. Sorry I missed this.

Reply:
Sorry I missed it as well.